docs/RaidbossGuide.md

```diff
@@ -446,6 +454,46 @@ This deprecation applies only to work intended for the cactbot repository.)
 Use of canned conditions and responses is recommended where possible, although
 given Square's extremely talented fight design team, it's not always going to *be* possible.
 
+## Outputs
+
+In order to reduce duplications across trigger sets,
+cactbot built a set of locale strings including some text wisely used between triggers.
+So we can easily reference to those translated text instead of translating/copy it one more time.
+
+A simple example using `outputStrings` and `Outputs` as below:
```

Comment:
I'm a little skeptical that this is a simple example, considering it requires a helper function that isn't listed in the example.

What about using `E9S Zero-Form Devouring Dark` as an example that combines Outputs and a custom output string, but without logic that's very complicated?

Reply:
Sure!